stock_quant_history/__manifest__.py

```diff
@@ -4,13 +4,15 @@
 {
     "name": "Stock Quant History",
     "summary": "Re-generate stock quants for given date",
-    "version": "14.0.1.0.0",
+    "version": "14.0.1.1.0",
     "license": "AGPL-3",
-    "author": "Pierre Verkest <pierreverkest84@gmail.com>, Odoo Community Association (OCA)",
+    "author": "Pierre Verkest <pierreverkest84@gmail.com>, Foodles, Stéphane Mangin "
+    "<stephane.mangin@foodles.com>, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/stock-logistics-reporting",
     "depends": ["stock"],
     "maintainers": [
         "petrus-v",
+        "Foodles",
```

Comment:
Not sure Foodles is a valid github user account, this gives merge access to the user on any PR related to that module.

If you want to add yourself, you are welcome to become maintainer of this module with me, as you wish !